From da7715a8ddc9ebb1e69cbfc6520a612346afacc0 Mon Sep 17 00:00:00 2001
From: montellese <montellese@kodi.tv>
Date: Mon, 20 Jan 2020 08:05:16 +0100
Subject: [PATCH] Revert "Print the mirror redirected to for easier debugging
 of of mirror issues"

This reverts commit de42ae8635eead2a678bc6c94738a03d55fb3e7b. It breaks
downloading dependencies from any mirror which does not return the "Location"
HTTP header. Furthermore it invalidates the value of ERRORLEVEL which now
contains the return code of findstr instead of from wget.
---
 project/BuildDependencies/scripts/get_formed.cmd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/project/BuildDependencies/scripts/get_formed.cmd b/project/BuildDependencies/scripts/get_formed.cmd
index 92fcb80a8c81..41f40510acd3 100644
--- a/project/BuildDependencies/scripts/get_formed.cmd
+++ b/project/BuildDependencies/scripts/get_formed.cmd
@@ -64,7 +64,7 @@ IF EXIST %1 (
 ) ELSE (
   CALL :setSubStageName Downloading %1...
   SET DOWNLOAD_URL=%KODI_MIRROR%/build-deps/win32/%1
-  %WGET% -S --quiet --tries=5 --retry-connrefused --waitretry=2 --show-progress "!DOWNLOAD_URL!" 2>&1 | findstr /L /I "Location:"
+  %WGET% --quiet --tries=5 --retry-connrefused --waitretry=2 --show-progress "!DOWNLOAD_URL!" 2>&1
   REM Apparently there's a quirk in cmd so this means if error level => 1
   IF ERRORLEVEL 1 (
     ECHO %1^|Download of !DOWNLOAD_URL! failed >> %FORMED_FAILED_LIST%
